src/cascadia/TerminalApp/defaults.json

```diff
@@ -0,0 +1,216 @@
+        },
+        {
+            "name": "Vintage",
+            "foreground": "#C0C0C0",
+            "background": "#000000",
+            "black": "#000000",
+            "red": "#800000",
+            "green": "#008000",
+            "yellow": "#808000",
+            "blue": "#000080",
+            "purple": "#800080",
+            "cyan": "#008080",
+            "white": "#C0C0C0",
+            "brightBlack": "#808080",
+            "brightRed": "#FF0000",
+            "brightGreen": "#00FF00",
+            "brightYellow": "#FFFF00",
+            "brightBlue": "#0000FF",
+            "brightPurple": "#FF00FF",
+            "brightCyan": "#00FFFF",
+            "brightWhite": "#FFFFFF"
+        },
+        {
+            "name": "One Half Dark",
+            "foreground": "#DCDFE4",
+            "background": "#282C34",
+            "black": "#282C34",
+            "red": "#E06C75",
+            "green": "#98C379",
+            "yellow": "#E5C07B",
+            "blue": "#61AFEF",
+            "purple": "#C678DD",
+            "cyan": "#56B6C2",
+            "white": "#DCDFE4",
+            "brightBlack": "#5A6374",
+            "brightRed": "#E06C75",
+            "brightGreen": "#98C379",
+            "brightYellow": "#E5C07B",
+            "brightBlue": "#61AFEF",
+            "brightPurple": "#C678DD",
+            "brightCyan": "#56B6C2",
+            "brightWhite": "#DCDFE4"
+        },
+        {
+            "name": "One Half Light",
+            "foreground": "#383A42",
+            "background": "#FAFAFA",
+            "black": "#383A42",
+            "red": "#E45649",
+            "green": "#50A14F",
+            "yellow": "#C18301",
+            "blue": "#0184BC",
+            "purple": "#A626A4",
+            "cyan": "#0997B3",
+            "white": "#FAFAFA",
+            "brightBlack": "#4F525D",
+            "brightRed": "#DF6C75",
+            "brightGreen": "#98C379",
+            "brightYellow": "#E4C07A",
+            "brightBlue": "#61AFEF",
+            "brightPurple": "#C577DD",
+            "brightCyan": "#56B5C1",
+            "brightWhite": "#FFFFFF"
+        },
+        {
+            "name": "Solarized Dark",
+            "foreground": "#839496",
+            "background": "#002B36",
+            "black": "#073642",
+            "red": "#DC322F",
+            "green": "#859900",
+            "yellow": "#B58900",
+            "blue": "#268BD2",
+            "purple": "#D33682",
+            "cyan": "#2AA198",
+            "white": "#EEE8D5",
+            "brightBlack": "#002B36",
+            "brightRed": "#CB4B16",
+            "brightGreen": "#586E75",
+            "brightYellow": "#657B83",
+            "brightBlue": "#839496",
+            "brightPurple": "#6C71C4",
+            "brightCyan": "#93A1A1",
+            "brightWhite": "#FDF6E3"
+        },
+        {
+            "name": "Solarized Light",
+            "foreground": "#657B83",
+            "background": "#FDF6E3",
+            "black": "#073642",

```

Comment:
dumb q but now can we reorder these to match VT order instead of .. this

Reply:
wait i'm a derp they already are

Reply:
i got chu